src/algebra/order/archimedean.lean

```diff
@@ -331,7 +331,8 @@ floor_ring.of_floor α (λ a, classical.some (exists_floor a))
   (λ z a, (classical.some_spec (exists_floor a) z).symm)
 
 /-- A linear ordered field that is a floor ring is archimedean. -/
-lemma floor_ring.archimedean (α) [linear_ordered_field α] [floor_ring α] : archimedean α :=
+@[priority 100] -- see Note [lower instance priority]
+instance floor_ring.archimedean (α) [linear_ordered_field α] [floor_ring α] : archimedean α :=
```

Comment:
Given this change, the `local attribute [instance] floor_ring.archimedean` in `to_interval_mod.lean` should no longer be needed.

Reply:
Good point, done.